tests/js/client/server_parameters/test-global-memory-limit-sjs.js

```diff
@@ -27,8 +27,8 @@
 
 if (getOptions === true) {
   return {
-    'query.global-memory-limit': "6000000",
-    'query.memory-limit': "5000000"
+    'query.global-memory-limit': "20000000",
+    'query.memory-limit': "19000000"
```

Comment:
Did you just increase those numbers very high?
Or is this now close to the upper bound?
This is almost 4 times the memory we reported earlier.

Can we see which test is hitting those limits now?